iib/workers/tasks/dc_utils.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from iib.workers.tasks.utils import skopeo_inspect
+
+
+def is_image_dc(image):
+    """
+    Detect declarative config image.
+
+    We can have two types of image - SQLite and Declarative Config
+    Those can be distinguished by LABES.
+    Declarative config will have defined this LABEL: operators.operatorframework.io.index.configs.v1
+
+    :param str image: the pull specification of the container image (usually from_image)
+    :return: True if image is declarative config type, False otherwise
+    :rtype: bool
+    """
+    skopeo_output = skopeo_inspect(f'docker://{image}')
+    dc_image_label = 'operators.operatorframework.io.index.configs.v1'
+    return dc_image_label in skopeo_output['Labels']
```

Comment:
Can we shrink line 19 and 20 together?

Reply:
We could, however this I separated intentionally for better readability